utilities/must_gather.py

```diff
@@ -3,12 +3,18 @@
 
 from pyhelper_utils.shell import run_command
 
-from utilities.constants import TIMEOUT_15MIN
+from utilities.constants import TIMEOUT_15MIN, TIMEOUT_20MIN
 
 
 def run_must_gather(
-    image_url="", target_base_dir="", script_name="", flag_names="", timeout=f"{TIMEOUT_15MIN}s", since=None
-):
+    image_url: str = "",
+    target_base_dir: str = "",
+    script_name: str = "",
+    flag_names: str = "",
+    timeout: str = f"{TIMEOUT_15MIN}s",
+    since: str | None = None,
+    run_command_timeout: int = TIMEOUT_20MIN,
```

Comment:
just a nit: do you think we need to have it as argument? not sure if someone is going to pass it...

Reply:
I think so. My guess it, for OCP gather you may need to adjust it at some point.